src/main/java/org/folio/rest/impl/EholdingsTitlesImpl.java

```diff
@@ -0,0 +1,110 @@
+package org.folio.rest.impl;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.folio.config.RMAPIConfigurationServiceCache;
+import org.folio.config.RMAPIConfigurationServiceImpl;
+import org.folio.config.api.RMAPIConfigurationService;
+import org.folio.http.ConfigurationClientProvider;
+import org.folio.rest.annotations.Validate;
+import org.folio.rest.aspect.HandleValidationErrors;
+import org.folio.rest.converter.TitleConverter;
+import org.folio.rest.jaxrs.model.TitlePostRequest;
+import org.folio.rest.jaxrs.resource.EholdingsTitles;
+import org.folio.rest.model.OkapiData;
+import org.folio.rest.model.Sort;
+import org.folio.rest.util.ErrorUtil;
+import org.folio.rest.validator.HeaderValidator;
+import org.folio.rest.validator.TitleParametersValidator;
+import org.folio.rmapi.RMAPIService;
+import org.folio.rmapi.exception.RMAPIServiceException;
+
+import javax.ws.rs.core.Response;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+public class EholdingsTitlesImpl implements EholdingsTitles {
+
+  private static final String GET_TITLES_ERROR_MESSAGE = "Failed to retrieve titles";
+
+  private final Logger logger = LoggerFactory.getLogger(EholdingsConfigurationImpl.class);
```

Comment:
this logger should be getLogger(EholdingsTitlesImpl.class) instead?

Reply:
@sduvvuri-ebsco this is still not corrected